File: Iniciante/1052 - Mês/Python/1052.py
def main():
    value = int(input())
    months = create_calendar()
    month = identify_month(value, months)
    print(month)


def create_calendar():
    months = {
        1: "January",
        2: "February",
        3: "March",
        4: "April",
        5: "May",
        6: "June",
        7: "July",
        8: "August",
        9: "September",
        10: "October",
        11: "November",
        12: "December",
    }
    return months


def identify_month(value, months):
    return months.get(value)


if __name__ == "__main__":
    main()

File: Iniciante/1052 - Mês/JavaScript/1052.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const value = parseInt(input);
  const months = createCalendar();
  const month = identifyMonth(value, months);
  console.log(month);
}

function createCalendar() {
  const months = new Map();

  months.set(1, "January");
  months.set(2, "February");
  months.set(3, "March");
  months.set(4, "April");
  months.set(5, "May");
  months.set(6, "June");
  months.set(7, "July");
  months.set(8, "August");
  months.set(9, "September");
  months.set(10, "October");
  months.set(11, "November");
  months.set(12, "December");

  return months;
}

function identifyMonth(value, months) {
  return months.get(value);
}

main();
